.circleci/manage-test-db.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
```

Comment:
I'm in the process of reviewing all the changes, but what happens if I run the integration tests directly? Does it depend on running this script first?

Reply:
Nope, when you run directly it doesn't create a test database for you, that's just for CircleCI.

It'll use whatever you set for the `<engine>_DATABASE` environment variable (assuming you're using the default config, otherwise you can of course override it to whatever you need).

For the docker databases, it does however run the `wait-for-db.sh` script which relies on `docker` and `netcat` being installed.

Reply:
btw - the way I was doing this locally is create a file `.env` at the root of the repo containing the env vars:

```
SNOWFLAKE_ACCOUNT=<redacted>
SNOWFLAKE_WAREHOUSE=small
SNOWFLAKE_DATABASE=erin
SNOWFLAKE_USER=<redacted>
SNOWFLAKE_PASSWORD=<redacted>

DATABRICKS_CATALOG=erin
DATABRICKS_SERVER_HOSTNAME=<redacted>
DATABRICKS_HTTP_PATH=<redacted>
DATABRICKS_ACCESS_TOKEN=<redacted>

REDSHIFT_HOST=<redacted>
REDSHIFT_PORT=5439
REDSHIFT_USER=<redacted>
REDSHIFT_PASSWORD=<redacted>
REDSHIFT_DATABASE=erin

BIGQUERY_KEYFILE=<redacted>
```

VSCode picks this up automatically so no extra config needed if running tests from VSCode.

To run the tests for eg Snowflake, you can do the following:

```
$ export $(cat .env | xargs)
$ make snowflake-test
```

For the docker databases it's even easier, just run:

```
$ make trino-test
```

Which will ensure the correct dependencies are installed, the containers are started and available and then kick off the tests